test/e2e/common/security_context.go

```diff
@@ -366,3 +367,20 @@ var _ = framework.KubeDescribe("Security Context", func() {
 		})
 	})
 })
+
+// waitForFailure waits for pod to fail.
+func waitForFailure(c *framework.PodClient, name string, timeout time.Duration) {
+	f := c.f
```

Comment:
> /b/f/w/test/e2e/common/security_context.go:373:8: c.f undefined (cannot refer to unexported field or method f

might have to pass the framework pointer explicitly, as the field `f` in the pod client is private

Reply:
@neolit123 Thanks for pointing it out here!
I am fixing and testing it locally now, will update this PR soon after that.